package.json

```diff
@@ -151,6 +152,7 @@
     "nodemon": "^2.0.15",
     "pino-pretty": "^5.1.0",
     "prettier": "^2.5.1",
+    "qrcode": "^1.5.0",
```

Comment:
not sure if we want to add additional packages for debug scripts. This can be done in an external script/project

Reply:
This seemed acceptable since we are doing the same [here](https://github.com/GaloyMoney/galoy/blob/main/src/debug/create-custom-invoices.ts#L8) with the `bip66` package, but now that you mentioned this that's probably just leftover.

Should we just remove them from `package.json` and add a step to the instructions of each script to install those packages manually?
Not sure I understand what you mean by "external script/project". Are you suggesting to wrap up the debug scripts in an npm package?

Reply:
I guess the goal is to move using `monorepo` @dolcalmi ?